README.md

```diff
@@ -114,6 +114,21 @@ There are two options when managing DNS records, manually or ExternalDNS.
 
 Assuming you already have everything needed to install the apps, this is what you need to do.
 
+> [!NOTE]
```

Comment:
I'm missing a section giving an overview of Apps. This is stuff that we already know and have in our DNA, which is why it's so hard to write it. Someone completely new to the project should first learn that:

- Welkin follows the "configuration as code" principle and expects all configuration to be in a so-called configuration folder.
- Welkin strongly recommends to have this configuration folder in `.git`. Apps may be a submodule, so that the configuration folder is the source of truth for how an environment is set up.
- Welkin Apps' functionality is accessed threw the `./bin/ck8s` command-line tool.

Makes sense?

Reply:
Pushed a change. Is that satisfactory? Hard to know where to exactly place it.